3_build.R

```diff
@@ -13,38 +13,28 @@ p3_targets <- list(
   
   # Add states
   # TODO: groups don't seem to actually be working
-  tar_target(
-    g_conus_state_svg,
-    add_grp(out_svg = "3_build/tmp/g_conus_state.svg", 
-            in_svg = root_svg, 
-            grp_nm = 'conus-states', trans_x = 0, trans_y = 0),
-    format = "file"
-  ),
-  
   tar_target(
     state_paths_svg, 
-    add_child_paths(out_svg = "3_build/tmp/state_paths.svg",
-                    in_svg = g_conus_state_svg,
-                    paths = p2_conus_states_paths,
-                    path_nms = sprintf('state-%s', p2_conus_states_names)),
+    add_poly_group_to_svg(
+      out_svg = "3_build/tmp/state_paths.svg",
+      in_svg = root_svg,
+      grp_id = "conus-states",
+      paths = p2_conus_states_paths,
+      path_ids = p2_conus_states_names,
+      path_class = "state"),
```

Comment:
This class seems to be getting lost. When I run the pipeline I'm seeing `class="NA"` on state paths

Reply:
The first state has `class="state"` but subsequent paths do not

Reply:
The trick was that I needed to repeat the values passed in here!